lib/morse_translator.rb

```diff
@@ -38,6 +40,75 @@ def initialize
                     "9" => "----.",
                     "0" => "-----",
                     " " => " "}
+
+
   end
 
+  def translate(string)
+    string_array = []
+    holder = string.downcase.chars
+    holder.each do |char|
+      dictionary.each do |key,val|
+        if char == key
+          string_array << val
+        end
+      end
+    end
+    string_array.join("")
+  end
+
+  def morse_to_eng(string)
```

Comment:
this is soooooo close!!!  I think you could make it work by updating line 62 to be `holder = string.split(" ")`.  But you are definitely on the right track here!